controllers/operatorpolicy_controller.go

```diff
@@ -1356,7 +1356,9 @@ func (r *OperatorPolicyReconciler) musthaveInstallPlan(
 			}
 		}
 
-		relatedInstallPlans = append(relatedInstallPlans, existingInstallPlanObj(&ownedInstallPlans[i], phase))
+		complianceConfig := policy.Spec.ComplianceConfig.UpgradesAvailable
```

Comment:
Could you set this once outside of the loop so it's not redefined every loop iteration and the code below can reference it as well?